test/e2e/framework/volume_util.go

```diff
@@ -473,24 +481,24 @@ func TestVolumeClient(client clientset.Interface, config VolumeTestConfig, fsGro
 	}
 	if !NodeOSDistroIs("windows") {
 		if fsGroup != nil {
-			By("Checking fsGroup is correct.")
+			ginkgo.By("Checking fsGroup is correct.")
 			_, err = LookForStringInPodExec(config.Namespace, clientPod.Name, []string{"ls", "-ld", "/opt/0"}, strconv.Itoa(int(*fsGroup)), time.Minute)
 			ExpectNoError(err, "failed: getting the right privileges in the file %v", int(*fsGroup))
 		}
 
 		if fsType != "" {
-			By("Checking fsType is correct.")
+			ginkgo.By("Checking fsType is correct.")
 			_, err = LookForStringInPodExec(config.Namespace, clientPod.Name, []string{"grep", " /opt/0 ", "/proc/mounts"}, fsType, time.Minute)
 			ExpectNoError(err, "failed: getting the right fsType %s", fsType)
 		}
 	}
 }
 
-// InjectHtml insert index.html with given content into given volume. It does so by
+// InjectHTML insert index.html with given content into given volume. It does so by
```

Comment:
Um...  a little piece of advice, please change `insert` to `inserts`

Reply:
done.